manifests/d/DearMob&Digiarty,Inc/DearMobiPhoneManager/6.0.0.0/DearMob&Digiarty,Inc.DearMobiPhoneManager.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: DearMob&Digiarty,Inc.DearMobiPhoneManager
+PackageVersion: 6.0.0.0
+PackageLocale: en-US
+Publisher: DearMob & Digiarty, Inc.
+PackageName: DearMob iPhone Manager
+License: Copyright © 2022 Digiarty Software, Inc. All Rights Reserved.

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate